aea/aea_builder.py

```diff
@@ -1366,17 +1369,35 @@ def _populate_contract_registry(self):
                 continue
             logger.debug(f"Registering contract {configuration.public_id}")
 
-            path = Path(
-                configuration.directory, configuration.path_to_contract_interface
-            )
-            with open(path, "r") as interface_file:
-                contract_interface = json.load(interface_file)
-
+            paths = {}  # type: Dict[str, str]
```

Comment:
@marcofavorito we need to allow for a dictionary of contract interface paths to be specified; one for each ledger identifier in order to support multi ledger contract packages.